spec/models/recipe_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I like this advice. If you have the `shoulda-matchers` gem you might be able to use

```ruby
it { is_expected_to belong_to(:user) }
```

I'll leave it up to you to decide whether that test has value for you. If it doesn't, how about deleting this file?